TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/ExemptionReason.cshtml.cs

```diff
@@ -1,23 +1,41 @@
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using TeachingRecordSystem.Core.DataStore.Postgres;
 
 namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.EditInduction;
 
-[Journey(JourneyNames.EditInduction), RequireJourneyInstance]
+[Journey(JourneyNames.EditInduction), ActivatesJourney, RequireJourneyInstance]
```

Comment:
shouldn't this page be changed as part of the select exemption reason ticket?

Reply:
On Slack you asked me to put all the links onto the Induction details tab and complete the journey logic, so I added the link to the exemption reason page and did the minimum on it to be able to test the journey